threestudio/data/uncond.py

```diff
@@ -314,10 +316,11 @@ def collate(self, batch) -> Dict[str, Any]:
         # Importance note: the returned rays_d MUST be normalized!
         rays_o, rays_d = get_rays(directions, c2w, keepdim=True)
 
-        proj_mtx: Float[Tensor, "B 4 4"] = get_projection_matrix(
+        self.proj_mtx: Float[Tensor, "B 4 4"] = get_projection_matrix(
             fovy, self.width / self.height, 0.1, 1000.0
         )  # FIXME: hard-coded near and far
```

Comment:
I noticed we have values ranging from 0.1 to 1000.0 in the following code. How about updating them all to a more consistent range, like 0.01 to 100.0? What do you think?